server/src/modules/mixin/exchange/exchange.repository.ts

```diff
@@ -92,6 +98,14 @@ export class ExchangeRepository {
     return await this.spotOrderRepository.update({ orderId }, { updatedAt });
   }
 
+  async updateSpotOrderLimitFilled(orderId: string, limitFilled: string) {
+    return await this.spotOrderRepository.update({ orderId }, { limitFilled });
+  }
+
+  async updateSpotOrderReceivedAmount(orderId: string, receiveAmount: string) {
+    return await this.spotOrderRepository.update({ orderId }, { receiveAmount });
+  }
+
   async updateSpotOrderApiKeyId(orderId: string, apiKeyId: string) {
     return await this.spotOrderRepository.update({ orderId }, { apiKeyId });
   }
```

Comment:
Isn't it better defining more general function like `updateSpotOrderDetails`, which receives partial object of the entire spot order details?

Reply:
@leric7 
There were already numerous separate CRUD methods for simple operations. I've just followed this convention. However, as requested, I've consolidated the methods I added into a more generic one called "updateByOrderId."

Reply:
@leric7 
There were already numerous separate CRUD methods for simple operations in this service. I've just followed this convention. However, as requested, I've replaced my methods with a more generic one called "updateSpotOrderDetails."